### ISSUE TITLE
Hara time: Wrong number of args (2) passed to: map/from-map

### ISSUE THREAD

**Issue by AdamFrey** [zcaudate/hara#28]
On Hara 2.4.5, with `(t/default-type) => clojure.lang.PersistantArrayMap`

A simple call to format
```
(t/format (t/now) "HH MM dd")
```

throws
```
   Wrong number of args (2) passed to: map/from-map

                  AFn.java:  429  clojure.lang.AFn/throwArity
                  AFn.java:   36  clojure.lang.AFn/invoke
java_text_simpledateformat.clj:   49  hara.time.data.format.java-text-simpledateformat/eval37852/fn
              MultiFn.java:  238  clojure.lang.MultiFn/invoke
                format.clj:   68  hara.time.data.format/format
                format.clj:   44  hara.time.data.format/format
                format.clj:   60  hara.time.data.format/format
                format.clj:   44  hara.time.data.format/format

```

It looks like these two calls to `map/from-map` are only passing 2 arguments not 3. https://github.com/zcaudate/hara/blob/master/src/hara/time/data/format/java_text_simp

**Comment by zcaudate** [comment]
oh crap. I'll take a look at it.

**Comment by zcaudate** [comment]
it's quite an easy fix. for now, can you patch: https://github.com/zcaudate/hara/blob/master/src/hara/time/data/format/java_text_simpledateformat.clj, lines 45 and 49:

```clojure
(time/-format formatter (map/from-map m {:type Calendar} opts) opts)
                                                                                                        |----- add extra `opts`
```

do a `lein repack install` on your local machine.

I'll make a release soon.

**Comment by zcaudate** [comment]
the fix has been done in 2.4.6
